ui/src/reusable_ui/components/wizard/WizardButtonBar.tsx

```diff
@@ -10,6 +10,7 @@ interface Props {
   lastStep?: boolean
   onClickPrevious: () => void
   onClickNext: () => void
+  testId?: {previousLabel: string; nextLabel: string}
```

Comment:
You should better use one test-id and derive the button IDs from it. It could have a reasonable default, such as `wizard-buttons`. As a UX invariant, there is always just one wizard on the page, the test could also stick to `wizard-button-bar` CSS class and button labels.